pkg/credentials/credentials.go

```diff
@@ -8,30 +8,46 @@ import (
 	"regexp"
 
 	"github.com/aws/aws-sdk-go/aws/credentials"
-	"github.com/commitdev/zero/internal/config/globalconfig"
+	"github.com/commitdev/zero/internal/util"
 )
 
-func AwsCredsPath() string {
+type AWSResourceConfig struct {
+	AccessKeyID     string `yaml:"accessKeyId,omitempty" env:"AWS_ACCESS_KEY_ID,omitempty"`
```

Comment:
Why "yaml"? If this annotation is just to be used by the `ReflectStructValueIntoMap` function we might as well use our own identifier so it's clear that it's not supposed to be used in some other way to generate yaml.

Reply:
No particular reason, mostly just because I reused the struct from before (global creds)
what should we change it to, should we just call it `zero`?